src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -562,7 +562,7 @@ protected function isGrantedOr403($permission)
         if ($permission) {
             $grant = sprintf('%s.%s.%s', $this->config->getBundlePrefix(), $this->config->getResourceName(), $permission);
 
-            if (!$this->get('sylius.authorization_checker')->isGranted($grant)) {
+            if (!$this->get('security.authorization_checker')->isGranted($grant)) {
```

Comment:
@aramalipoor I agree, but I was waiting for feedback from @pjedrzejewski. Pawel, what do you think? I'm afraid this work could become outdated quite quickly...